src/main/scala/slamdata/SbtSlamData.scala

```diff
@@ -273,6 +287,70 @@ object SbtSlamData extends AutoPlugin {
   private def transferScripts(baseDir: File, srcs: String*) =
     srcs.foreach(src => transfer(src, baseDir / "scripts" / src, Set(OWNER_EXECUTE)))
 
+  private def slamdataVersionCheck = Def.task {
+    import scala.sys.process._
+
+    val logger: Logger = streams.value.log
+    def relog(line: String): Unit = {
+      val level = """\[([^]]*)\].*""".r
+      line match {
+        case level("debug") => logger.debug(line)
+        case level("info") => logger.info(line)
+        case level("warn") => logger.warn(line)
+        case level("error") => logger.error(line)
+        case level("success") => logger.success(line)
+        case _ => logger.info(line)
+      }
+    }
+    val logToSbt = ProcessLogger(relog _)
+
+    val getCsbt = "wget -O csbt https://github.com/coursier/sbt-extras/raw/master/sbt"
```

Comment:
Probably better to use `curl` here, since a lot of systems actually don't have `wget` preinstalled.